plugins/webp-uploads/picture-element.php

```diff
@@ -146,7 +146,7 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 	remove_filter( 'wp_calculate_image_srcset_meta', $filter );
 
 	return sprintf(
-		'<picture class="%s">%s%s</picture>',
+		'<picture class="%s" style="display: contents;">%s%s</picture>',
```

Comment:
Instead of adding style in all picture elements can we add custom CSS for element for gallery block?

something like: `.block picture { display: contents; }`

Reply:
Thanks @mukeshpanchal27. Can you please provide more context for making these changes to `wp:gallery` and not here? Any specific reason?

Reply:
I believe it makes sense to apply this globally to all `picture` elements specifically generated by this plugin, as done here, as it will essentially ignore the `picture` for styling purposes. This is what we want because we only care about the `img`.

Reply:
Gotcha!